README.md

```diff
@@ -3,50 +3,60 @@
 Bergamot translator provides a unified API for ([Marian NMT](https://marian-nmt.github.io/) framework based) neural machine translation functionality in accordance with the [Bergamot](https://browser.mt/) project that focuses on improving client-side machine translation in a web browser.
 
 ## Build Instructions
+
 ### Build Natively
-```
-$ git clone https://github.com/browsermt/bergamot-translator
-$ cd bergamot-translator
-$ mkdir build
-$ cd build
-$ cmake ../
-$ make -j
 
+```bash
+git clone https://github.com/browsermt/bergamot-translator
+cd bergamot-translator
+mkdir build
+cd build
+cmake ../
+make -j
 ```
 
-### Build on WASM
-To compile on WASM, first download and Install Emscripten using following instructions:
-```
-Get the latest sdk: git clone https://github.com/emscripten-core/emsdk.git
-Enter the cloned directory: cd emsdk
-Install the lastest sdk tools: ./emsdk install latest
-Activate the latest sdk tools: ./emsdk activate latest
-Activate path variables: source ./emsdk_env.sh
-```
+### Build WASM
+
+To compile WASM, first download and Install Emscripten using following instructions:
+
+1. Get the latest sdk: `git clone https://github.com/emscripten-core/emsdk.git`
+2. Enter the cloned directory: `cd emsdk`
+3. Install the lastest sdk tools: `./emsdk install latest`
+4. Activate the latest sdk tools: `./emsdk activate latest`
+5. Activate path variables: `source ./emsdk_env.sh`
+
 After the successful installation of Emscripten, perform these steps:
+
+```bash
+git clone https://github.com/browsermt/bergamot-translator -b wasm
```

Comment:
`-b wasm` should be removed from here.

Reply:
Currently, only the wasm branch contains the dummy api. Remove it when/before this code goes into the main branch again?

Reply:
With the current change, the build instructions become wrong. However, I am fine with the proposed suggestion.